.mergify.yml

```diff
@@ -301,39 +303,39 @@ pull_request_rules:
               - or:
                   - label=ci/skip/e2e
                   - and:
-                      - "status-success=ci/centos/k8s-e2e-external-storage/1.29"
                       - "status-success=ci/centos/k8s-e2e-external-storage/1.30"
                       - "status-success=ci/centos/k8s-e2e-external-storage/1.31"
-                      - "status-success=ci/centos/mini-e2e-helm/k8s-1.29"
+                      - "status-success=ci/centos/k8s-e2e-external-storage/1.32"
                       - "status-success=ci/centos/mini-e2e-helm/k8s-1.30"
                       - "status-success=ci/centos/mini-e2e-helm/k8s-1.31"
-                      - "status-success=ci/centos/mini-e2e/k8s-1.29"
+                      - "status-success=ci/centos/mini-e2e-helm/k8s-1.32"
                       - "status-success=ci/centos/mini-e2e/k8s-1.30"
                       - "status-success=ci/centos/mini-e2e/k8s-1.31"
+                      - "status-success=ci/centos/mini-e2e/k8s-1.32"
                       - "status-success=ci/centos/upgrade-tests-cephfs"
                       - "status-success=ci/centos/upgrade-tests-rbd"
     actions:
       queue:
         name: default
       delete_head_branch: {}
 
-  - name: backport patches to release-v3.12 branch
+  - name: backport patches to release-v3.13 branch
     conditions:
       - base=devel
-      - label=backport-to-release-v3.12
+      - label=backport-to-release-v3.13
     actions:
       backport:
         branches:
-          - release-v3.12
+          - release-v3.13
 
-  - name: backport patches to release-v3.13 branch
+  - name: backport patches to release-v3.14 branch
     conditions:
       - base=devel
-      - label=backport-to-release-v3.13
+      - label=backport-to-release-v3.14
     actions:
       backport:
         branches:
-          - release-v3.13
+          - release-v3.14
```

Comment:
can you please update https://github.com/ceph/ceph-csi/blob/devel/.github/workflows/pull-request-commentor.yaml#L21 to include 3.14 as well

Reply:
Done, please take a look again